content_scripts/approvals.js

```diff
@@ -1,34 +0,0 @@
-EnhancedPullRequest.approvals = {
```

Comment:
You'll need to delete the `on an issue comment with an approval message` block in spec/conversation_spec.rb